doc/source/whatsnew/v1.5.0.rst

```diff
@@ -121,7 +121,7 @@ Datetimelike
 
 Timedelta
 ^^^^^^^^^
--
+- Now raising on missing or invalid kwargs (:issue:`43764`)
```

Comment:
Since this is a regression in 1.4, im pretty sure this note goes in 1.4.1, can double-check with @simonjayhawkins

Reply:
actually move to 1.4 as we haven't released yet